### PR TITLE
fixed issue -16293 Pagination NOT showing properly in Twenty Seventee…

### PR DESCRIPTION
**Pull request by adnanlimdiwala**
@mikejolley 

I have fixed pagination not showing properly in dark version of the Twenty Seventeen theme. 

Can you please check my PR and let us know if any further changes required.

#16293 

Thanks,

### CODE REVIEW

File: assets/css/twenty-seventeen.scss [hunk 1]
```diff
@@ -102,7 +102,17 @@
...
+	.woocommerce-pagination {
+		span.page-numbers.current {
+			background-color: #fff;
+		}
+		.next.page-numbers,.prev.page-numbers,a.page-numbers,span.page-numbers{
+			background: #505050;
+		}	
+		
+	}
+}
```
Comment (claudiulodro) [inline assets/css/twenty-seventeen.scss]: - This doesn't keep the color fading and switching effect that happens when you hover over a button. Can you make the number buttons have the same effect as the arrow buttons when hovered?
- This code should go in the dark colors style section starting at line 1008. That way all of the dark colors style is in the same place.

Comment (claudiulodro) [inline assets/css/twenty-seventeen.scss]: I'm pretty sure all you have to do is make the text color darker for the non-hovered number buttons to fix the issue.

**Comment by adnanlimdiwala** [comment]
@claudiulodro 
I have update code now can you please check my PR and let us know if any further changes required.

**Comment by claudiulodro** [comment]
Good effort. I think you're making this too complicated, though. See #16330

**Pull request closed without merging**